app/signals/apps/api/views/ml_tool_proxy.py

```diff
@@ -8,6 +8,12 @@
 
 from signals.apps.api.ml_tool.client import MLToolClient
 from signals.apps.signals.models import Category
+import pickle
```

Comment:
Misschien is het mooi (om conflicten te voorkomen) dat we een nieuwe view onderbrengen in signals/apps/classification/views.py.